wicket-core/src/test/java/org/apache/wicket/markup/MarkupParserTest.java

```diff
@@ -67,31 +67,31 @@ void tagParsing() throws Exception
 		final ComponentTag aOpen = (ComponentTag)markupStream.next();
 
 		log.info("", aOpen);
-		assertTrue(aOpen.getName().equals("a"));
+		assertEquals(aOpen.getName(), "a");
```

Comment:
These are valid changes :))))

Reply:
What is your opinion about this refactoring, please? Why do you think there are many cases like `assertTrue(x.equals(y))` and `assertFalse(x.equals(y))` or `assertTrue(!x.equals(y))`, though `assertEquals` and `assertNotEquals` are more readable/intuitive and can provide better error messages in case of failures?

Reply:
I can't say for sure

I guess someone just overlooked _the right way_ :)))